dot/core/service_integration_test.go

```diff
@@ -197,6 +318,11 @@ func TestService_HasKey_UnknownType(t *testing.T) {
 	cfg := &Config{
 		Keystore: ks,
 	}
+
+	ctrl := gomock.NewController(t)
+	digestHandler := NewMockDigestHandler(ctrl)
+	cfg.DigestHandler = digestHandler
```

Comment:
If we don't configure it, I think leaving to nil will work too so we could remove those new lines

Reply:
good call!